.eslintrc

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["webpack", "prettier"],
+  "extends": ["webpack"],
```

Comment:
Not we use `prettier` and `eslint` as separate tools. Why:
- better integration with other files like `yml`, `md`, `css` and etc
- prettier inside eslint can break code https://github.com/prettier/eslint-config-prettier#arrow-body-style-and-prefer-arrow-callback